webapp/src/Service/ExternalContestSourceService.php

```diff
@@ -1023,17 +1028,22 @@ protected function validateAndUpdateProblem(Event $event, EventData $data): void
         $this->removeWarning($event->type, $data->id, ExternalSourceWarning::TYPE_ENTITY_NOT_FOUND);
 
         $toCheckProblem = [
-            'name'      => $data->name,
-            'timelimit' => $data->timeLimit,
+            'name' => $data->name,
         ];
 
+        if ($data->timeLimit !== null) {
+            $toCheckProblem['timelimit'] = $data->timeLimit;
+        }
+
+        /* Disable as PC2 can have 2 problems with the same label
```

Comment:
This is still a pending discussion, and it won't happen at the finals so I'd rather not commit this for now.

Reply:
Taken the commit out.